test/test_sbom.py

```diff
@@ -90,3 +90,27 @@ def test_valid_swid_file(
     ):
         sbom_engine = SBOMManager(filename, sbom_type="swid")
         assert sbom_engine.scan_file() == swid_parsed_data
+
+    @pytest.mark.parametrize(
+        "filename, sbom_type, validate",
+        (
+            (os.path.join(SBOM_PATH, "swid_test.xml"), "spdx", True),
+            (os.path.join(SBOM_PATH, "swid_test.xml"), "cyclondedx", True),
+            (os.path.join(SBOM_PATH, "swid_test.xml"), "spdx", False),
+            (os.path.join(SBOM_PATH, "swid_test.xml"), "cyclondedx", False),
+            (os.path.join(SBOM_PATH, "cyclonedx_test.xml"), "spdx", True),
+            (os.path.join(SBOM_PATH, "cyclonedx_test.xml"), "swid", True),
+            (os.path.join(SBOM_PATH, "cyclonedx_test.xml"), "spdx", False),
+            (os.path.join(SBOM_PATH, "cyclonedx_test.xml"), "swid", False),
+            (os.path.join(SBOM_PATH, "spdx_test.spdx.xml"), "cyclonedx", True),
+            (os.path.join(SBOM_PATH, "spdx_test.spdx.xml"), "swid", True),
+            (os.path.join(SBOM_PATH, "spdx_test.spdx.xml"), "cyclonedx", False),
+            (os.path.join(SBOM_PATH, "spdx_test.spdx.xml"), "swid", False),
+        ),
+    )
+    def test_invalid_xml(self, filename: str, sbom_type: str, validate: bool):
+        # Demonstrate that validation of XML file against schema results in no data
+        # if file does not match schema or if xml data is parsed against wrong type of sbom
+        # (indicated by validate being set to False)
```

Comment:
It might be better to use docstrings instead of comments according to [PEP8](https://www.python.org/dev/peps/pep-0008/#documentation-strings).

Reply:
@BreadGenie - Thanks I have fixed my conventional commits formatting error and I will; update comments to PEP8 convention later and resubmit pull request